ulauncher/search/apps/AppResultItem.py

```diff
@@ -1,42 +1,87 @@
+from os.path import basename
+import gi
+gi.require_version('Gio', '2.0')
+# pylint: disable=wrong-import-position
+from gi.repository import Gio
+from ulauncher.utils.Settings import Settings
+from ulauncher.utils.fuzzy_search import get_score
 from ulauncher.api.shared.action.LaunchAppAction import LaunchAppAction
 from ulauncher.api.shared.item.ResultItem import ResultItem
 from ulauncher.search.QueryHistoryDb import QueryHistoryDb
 from ulauncher.search.apps.AppStatDb import AppStatDb
+from ulauncher.utils.image_loader import get_app_icon_pixbuf
+
+settings = Settings.get_instance()
+disable_desktop_filters = settings.get_property('disable-desktop-filters')
+blacklisted_dirs = settings.get_property('blacklisted-desktop-dirs')
+blacklisted_dirs_list = blacklisted_dirs.split(':') if blacklisted_dirs else []
```

Comment:
The values should probably be read in the body of the class, otherwise they are only set at import time, and never refreshed.

Reply:
Thank you :+1: Such an obvious one 🤦‍♂️